jormungandr/src/topology/process.rs

```diff
@@ -3,9 +3,12 @@ use crate::intercom::{NetworkMsg, PropagateMsg, TopologyMsg};
 use crate::settings::start::network::Configuration;
 use crate::utils::async_msg::{MessageBox, MessageQueue};
 use futures::SinkExt;
-use tokio::time::Interval;
+use std::time::Duration;
+use tokio::time::{Instant, Interval};
 use tokio_stream::StreamExt;
 
+const STUCK_NETWORK_WARNING: Duration = Duration::from_secs(60 * 5); // 5 min
```

Comment:
Wondering if this value should be configurable.

Reply:
Probably yes, It was just easier to set it up like this.